src/hooks/Trades.ts

```diff
@@ -17,35 +18,24 @@ function useAllCommonPairs(currencyA?: Currency, currencyB?: Currency): Pair[] {
     ? [wrappedCurrency(currencyA, chainId), wrappedCurrency(currencyB, chainId)]
     : [undefined, undefined]
 
+  const customBases = CUSTOM_BASES[chainId]
+  const customBasesA: Token[] = customBases && tokenA ? customBases[tokenA.address] : []
```

Comment:
can we do these like `CUSTOM_BASES[chainId]?.[tokenA?.address] ?? []`

Reply:
if tokenA is undefined, you're trying to index into an object with string keys by the key undefined